kolibri/core/assets/src/mixins/nav-components.js

```diff
@@ -37,5 +37,15 @@ export default {
         return this.isLearner || this.isCoach || this.isAdmin || this.isSuperuser;
       }
     },
+    filterIfSoUD(isSoUD) {
+      // Returns a filter predicate depending on if on SoUD
+      if (!isSoUD) {
+        // Don't filter anything if not on SoUD
+        return () => true;
+      } else {
+        const hiddenNavItemRoles = [UserKinds.COACH, UserKinds.ADMIN];
```

Comment:
Nice, I think this is the neatest way this could have been done.

Definitely agree with the Superuser being left there, as that's a device specific designation.